sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-data.js

```diff
@@ -118,18 +144,20 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
             // Check the top pixel and bottom pixel and make sure they have
             // the right color.
-            debug("Checking bottom pixel");
-            wtu.checkCanvasRect(gl, 0, 0, 1, 1, bottomColor, "shouldBe " + bottomColor);
-            debug("Checking top pixel");
-            wtu.checkCanvasRect(gl, 0, 1, 1, 1, topColor, "shouldBe " + topColor);
+            debug("Checking " + (flipY ? "top" : "bottom"));
+            wtu.checkCanvasRect(gl, 0, bottom, halfWidth, halfHeight, tl, "shouldBe " + tl);
```

Comment:
But the tests are all against RGBA8 framebuffer. These suite of tests only test loading image data from the DOM elements (<CANVAS>, <IMG>, <SVG>, <VIDEO>, etc) into a texture that is then rendered into the default framebuffer attached to another <CANVAS> element, which is RGBA8.